src/templates/appcodetails.js

```diff
@@ -661,7 +671,7 @@ class AppDetails extends Component {
           ]}
         />
 
-        {!isClaimedApp &&
+      {!isClaimedApp && showUpdateDialog &&
```

Comment:
Why the extra check? Does it matter for performance or so?

Reply:
actually, it is not needed but we might need it later on...